kong/plugins/prometheus/exporter.lua

```diff
@@ -156,12 +170,16 @@ local function collect()
                                         {res.workers_lua_vms[i].pid})
   end
 
+  -- force a manual sync of counter local state to make integration test working
+  counter_instance:sync()
```

Comment:
The way this plugin works is that the counter will be read only when the `collect` method is called. Is it possible to not sync periodically and only call `sync` before `collect` is called? It can potentially reduce the number of sync events.

Reply:
`sync` will only sync the local state on current worker to shared dict, to tell other worker to sync as well we need something like an event system and might be too heavy for this use case. `reduce the number of sync events` this is a good point, we can expose a "sync interval" via plugin config to let user match their prometheus scrape interval. i can do it in this PR if you feel okay with this approach.

Reply:
Got it. Thanks for the explanation. This doesn't need to be configurable via plugin configuration option right now (it would be part of a larger work to allow for dynamic changes to metrics and such counters).